### PR TITLE
Adding clear functionality to logs

### LINKED ISSUE #2: Clear current log set
At times, your log data set can become very large and you want to purge the current log history.

Add a clear button to purge the logs that match the current filters.

### PR DESCRIPTION
**Pull request by joshschumacher**
- Clear button in right side of filter bar will clear currently selected data (if on all namespace, all filters, it will clear all data, otherwise it will delete only data matching the selected namespace(s) and filter(s).
- Renamed output file to fn.logger.min.js
- Added unit tests for clear method

Fixes #2

### CODE REVIEW

File: src/fn.logger.js [hunk 1]
```diff
@@ -197,6 +197,24 @@ application.config(['$provide', function($provide) {
...
+
+      if (!_.isEmpty(namespaces)) {
+        query.namespace = namespaces;
+      }
+
+      if (!_.isEmpty(levels)) {
+        query.level = levels;
+      }
+
+      $delegate.datastore(query).remove();
```
Comment (jackofseattle) [inline src/fn.logger.js:215]: To help future proof this a little, lets double check that the remove method exists before we call it.

**Comment by jackofseattle** [comment]
Simple comment then LGTM

File: src/fn.logger.js (hunk 1 above)
Comment (joshschumacher) [inline src/fn.logger.js:215]: I like the idea of future proofing this for different datastores in the future but there's a lot of code that will need to be touched when we add in that ability.

The other calls that naively call chained methods are:

```
$delegate.datastore().distinct('namespace');
$delegate.datastore(query).order('time desc').get();
$delegate.datastore.insert({
```

Comment (joshschumacher) [inline src/fn.logger.js:215]: Going to merge this - we'll address custom datastore checking in #4

**Pull request merged**